packages/blobs/riscv64/thead/lpi4a-sysfan.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+#ENABLE FAN (PWM1)
+
+echo 1 > /sys/class/pwm/pwmchip0/export
+echo 1000000 > /sys/class/pwm/pwmchip0/pwm1/period
+echo 1000000 > /sys/class/pwm/pwmchip0/pwm1/duty_cycle
+echo 1 > /sys/class/pwm/pwmchip0/pwm1/enable
```

Comment:
CC @chainsx plz context to why is this needed instead of letting the board to handle the fans by itself, thanku~ ^-^

Reply:
This has already been defined in DTS.

Reply:
I think you should read this [PR](https://github.com/armbian/build/pull/5585) carefully.

Reply:
i do not see that in DTS:
* https://github.com/search?q=repo%3Achainsx%2Fthead-u-boot%20pwmchip0&type=code
* https://github.com/chainsx/thead-u-boot/blob/extlinux/dts/Kconfig

Can you elaborate?

Feel free to talk in chinese